docs/framework/network-programming/tls.md

```diff
@@ -54,19 +77,64 @@ For Windows Forms and other applications, see [How to: Target a Version of the .
 
 Use the following sections to verify you're not using a specific TLS or SSL version.
 
-## If your app targets .NET Framework 4.7 or later versions
+## If you must explicitly set a Security Protocol
+
+If you must explicitly set a security protocol instead of letting .NET or the OS pick the security protocol pick these protocols:
+
+- For .NET Framework 3.5: TLS 1.2
+- For .NET Framework 4.6.2 or later: TLS 1.3
+
+If you can't find specified protocols in enum, you can add those as an extension file. See below:
+
+`SslProtocolExtensions.cs`
+
+```c#
+namespace System.Security.Authentication
+{
+    public static class SslProtocolsExtensions
+    {
+        // For .NET Framework 3.5
+        public const SslProtocols Tls12 = (SslProtocols)3072;
+        // For .NET Framework 4.6.2 and later
+        public const SslProtocols Tls13 = (SslProtocols)12288;
```

Comment:
BTW did you test it? As far as I remember some people had problem even with the casting if running on older versions. It may work for you build (e.g. targeting) but it _may_ have problems when executed on older runtime. But I'm not sure how much we care. That should be rare as of now AFAIK.

Reply:
I tested it, and it's working fine, however we're generally talking about targeting version here, not the runtime version, so we should be fine.